modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -31,7 +31,10 @@ def submit
           pdf_data = get_pdf_data
           pdf_mapper_service(form_attributes, pdf_data, target_veteran).map_claim
 
-          get_benefits_documents_auth_token unless Rails.env.test?
+          evss_data = evss_mapper_service(auto_claim).map_claim
+          evss_service.submit(auto_claim, evss_data)
+
+          get_benefits_documents_auth_token
```

Comment:
@FonzMP  I deleted that other branch as it got a little too messy after some issues yesterday but I did not want your previous comments to disappear so adding a note here, the Test env checks are gone.